### PR TITLE
Updated readme: info about required ANDROID_HOME value

### PR DESCRIPTION
**Pull request by piotrek-buchman**

### CODE REVIEW

**Comment by masojus** [comment]
Something is up with the Travis CI build. Someone will need to take a look, @Geeber

File: README.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ repositories {
     mavenCentral()
 }
 dependencies {
-    compile 'io.keen:keen-client-api-java:5.2.0'
+    compile 'io.keen:keen-client-api-java:5.2.0'  // streaming 
+    compile 'io.keen:keen-client-api-query:5.2.0' // querying
```
Comment (masojus) [inline README.md]: This is only needed for query functionality, which is discussed here: https://github.com/keenlabs/KeenClient-Java/blob/35ae90dfbf4e9a02b9d60693f92bf18d3a9711bb/README.md#add-the-keen-query-client-package-to-your-build
So, I'm not sure if it should be placed in this section or not.

File: README.md [hunk 2]
```diff
@@ -40,6 +41,12 @@ Paste the following snippet into your pom.xml:
   <artifactId>keen-client-api-java</artifactId>
   <version>5.2.0</version>
 </dependency>
+
+<dependency>
```
Comment (masojus) [inline README.md]: Same comment as above.

File: README.md (hunk 1 above)
Comment (piotrek-buchman) [reply README.md]: Hi @masojus - thanks for the feedback. My intention was to emphasise that `io.keen:keen-client-api-query:5.2.0` is different dependency than `io.keen:keen-client-api-java:5.2.0`. Basing on your comment I moved it to a proper section of readme

File: README.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ repositories {
     mavenCentral()
 }
 dependencies {
-    compile 'io.keen:keen-client-api-java:5.2.0'
+    compile 'io.keen:keen-client-api-java:5.2.0'
```
Comment (Geeber) [inline README.md]: Nit: Please remove extra trailing space.

**Comment by masojus** [comment]
Sweet. Thanks for looking at the build issues @piotrek-buchman!

**Pull request merged**